### PR TITLE
Check null before process file

### PR DESCRIPTION
**Pull request by bammoo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,6 +15,11 @@ module.exports = function (processors, options) {
 
   stream._transform = function (file, encoding, cb) {
 
+    if (file.isNull()) {
+      this.push(file);
+      return cb();
```
Comment (w0rm) [inline index.js]: Can be written like this: `return cb(null, file)`. 

Please adjust to the coding style (no semicolons) and add a test.

**Comment by w0rm** [comment]
@bammoo thanks for the contribution, please fix the small issues and I'll be happy to merge.

**Comment by TrySound** [comment]
@w0rm You can add eslint to fail test on bad code style

**Comment by w0rm** [comment]
@TrySound thanks Captain, a PR with this would be very welcome!

**Comment by TrySound** [comment]
@w0rm Heh. It's not my code style :) I use [eslint-config-postcss](https://github.com/postcss/eslint-config-postcss) and [eslint-config-i-am-meticulous](https://github.com/MoOx/eslint-config-i-am-meticulous). They do not fit yours. Find another or write new.

**Comment by bammoo** [comment]
@w0rm Sorry for I have no idea to write such kind of test. I just encountered it when using [gulp-watch](https://github.com/floatdrop/gulp-watch) with pipe gulp-postcss.

**Comment by w0rm** [comment]
@bammoo it is simple, just write a null file into the transform stream and see if it comes out unchanged in data event. If you can't then I can do this on the weekend.

**Comment by w0rm** [comment]
@bammoo thanks!

**Pull request merged**